pandas/tests/io/test_encoding_aliases.py

```diff
@@ -0,0 +1,32 @@
+import pandas, os, nose
+
```

Comment:
The test suite is "short enough" that I believe you can run `pd.test(raise_warnings=())` and cover everything.  The way the test suite works is as follows:

The main test suite is `parsers.py`, which imports from `common.py` and a WHOLE TON of other modules in that same directory.  It is these imports that make it tricky to run individually.  If you look at the test classes, you see that they inherit from test classes like those in `common.py` to compose the test suites for each engine (`'c'` and `'python'`).